gremlin/agent_apparmor.profile

```diff
@@ -0,0 +1,100 @@
+#include <tunables/global>
+
+
+profile gremlin-agent flags=(attach_disconnected,mediate_deleted) {
+  #include <abstractions/base>
+
+  network inet tcp,
+  network inet udp,
+  network inet icmp,
+
+  deny network raw,
+
+  deny network packet,
+
+  file,
+  umount,
+
+  /entrypoint.sh rx,
+  /var/log/gremlin-api/** rwx,
+  /var/lib/gremlin/** rwx,
+  /var/log/gremlin/** rwx,
+
```

Comment:
We also read from `/etc/gremlin` for the config.yml file. Should we define that here?

Reply:
good catch